### PR TITLE
Latest projects: adding feature.

### PR DESCRIPTION
**Pull request by turizspace**
with some changes and screen shots . 
Screenshot (5)
Screenshot 2023-12-20 102953

### CODE REVIEW

**Comment by sajald77** [comment]
@turizspace I'm not sure what you've shown in the screenshot, it looks like a filter for `inactive projects`. Here are a few issues I have with it:
 1. I don't see any latest projects filter option there. 
 2. the list of project can simply use the projectCard, and render what is already being rendered in the landing page, that'll make things consistent, all you need to do is pass in project props to the project card component. 
 3. Also for the desktop view the layout needs a border outside of it, like for other views with a list. This is to keep things consistent.

**Review by sajald77 — CHANGES_REQUESTED** [review]
@turizspace I'm not sure what you've shown in the screenshot, it looks like a filter for inactive projects. Here are a few issues I have with it:

1. I don't see any latest projects filter option anymore, no way to go to the latest page
2. the list of project can simply use the projectCard, and render what is already being rendered in the landing page, that'll make things consistent, all you need to do is pass in project props to the project card component.
3. Also for the desktop view the layout needs a border outside of it, like for other views with a list. This is to keep things consistent.
4. some of the projects have scroll inside of it, because the content is too larger, in such cases let's have elipsis.

5. Also, can anything be done about pagination, we keep fetching 100's of projects, it's a waste of bandwidth for most users, as most people won't end up scrolling that much

**Comment by turizspace** [comment]
made these changes . added load more for pagination. and sort & filter on mobile view
Screenshot 2024-01-05 071906
Screenshot 2024-01-05 073925

**Review by sajald77 — CHANGES_REQUESTED** [review]
Good job there @turizspace . We're almost there.  only a couple of rounding off and then we're good. 

1. 
The border color and the font size and color for the description text should be updated as follows.

Screenshot 2024-01-10 at 10 01 49 AM


2. The empty images should be replaced with greyImage, simply use <ImageWithReload /> component, that'll do that for you automatically if there is no image or the image could not be loaded.


3. The loadmore button works fantastic, it just is not consistent with the rest in style, use

<Button variant='secondary' /> for this use case. We can use variants for using already themed components in the platform. 

Screenshot 2024-01-10 at 10 05 25 AM



4.  The text if too long seems to over flow, it should be set to two lines max and then have elipsis if not. You can use Chakra <Text /> component, and pass in, maxLines, and isTruncated parameters for an easy implementation.
Screenshot 2024-01-10 at 10 07 22 AM

**Comment by turizspace** [comment]
requested changes made.

**Comment by turizspace** [comment]
@sajald77 requested changes made .

File: src/pages/landing/projects/views/Latest.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import { Container, VStack, HStack, useBreakpointValue, Button, useDisclosure, Box } from '@chakra-ui/react';
+import { SimplePool, Event } from 'nostr-tools';
+import Noteslist from '../components/Noteslist';
+import { CardLayout } from '../../../../components/layouts';
+import { FilterDrawer } from '../../filters/mobile/FilterDrawer';
+import { MobileTopBar } from '../../filters/mobile/MobileTopBar';
+
+const GEYSERELAY = [
```
Comment (sajald77) [inline src/pages/landing/projects/views/Latest.tsx:9]: change it to GEYSER_RELAY_URL

File: src/pages/landing/projects/views/Latest.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import Noteslist from '../components/Noteslist';
+import { CardLayout } from '../../../../components/layouts';
+import { FilterDrawer } from '../../filters/mobile/FilterDrawer';
+import { MobileTopBar } from '../../filters/mobile/MobileTopBar';
+
+const GEYSERELAY = [
+  'wss://relay.geyser.fund',
+];
+
+export const Latest = () => {
```
Comment (sajald77) [inline src/pages/landing/projects/views/Latest.tsx:13]: Change name to LatestNostrProjects, to be more clear about what this is for.

File: src/pages/landing/projects/components/NoteCard.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+import { Text, Box, Flex, Link, useBreakpointValue } from '@chakra-ui/react';
+
+const ImageWithReload = ({ src, alt, ...rest }) => {
```
Comment (sajald77) [inline src/pages/landing/projects/components/NoteCard.tsx:4]: Image with Reload is already a component, that is present, you can simply import and use it, it has more functionality and handles the case of showing the grey image when image fails to load.
Please import and use the existing component.

I'm curious why you created a separate component here?

File: src/pages/landing/projects/components/NoteCard.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  );
+};
+
+interface Props {
+  content: string;
+  nostrich: string;
+  date: string;
+}
+
+export default function NoteCard({ content: rawContent, nostrich, date }: Props) {
```
Comment (sajald77) [inline src/pages/landing/projects/components/NoteCard.tsx:40]: Change name to NostrProjectCard, and we use ES6 style arrow functions, let's stick to that standard.

File: src/pages/landing/projects/components/Noteslist.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import { Event } from 'nostr-tools';
+import { Button, Flex } from '@chakra-ui/react';
+import NoteCard from './NoteCard.tsx';
+
+interface Props {
+  projects: Event[];
+}
+
+export default function NotesList({ projects }: Props) {
```
Comment (sajald77) [inline src/pages/landing/projects/components/Noteslist.tsx:10]: Change name to NostrProjectList, for readability and understandability of what is going on here.

**Review by sajald77 — CHANGES_REQUESTED** [review]
Great work man, Final small changes to the code for meeting coding standards, and we should be good.

File: src/pages/landing/projects/components/Noteslist.tsx (hunk 5 above)
Comment (turizspace) [reply src/pages/landing/projects/components/Noteslist.tsx]: @sajald77 done the renaming, and other standardization changes requested

**Pull request merged**